src/main/java/org/openjdk/jextract/impl/StructBuilder.java

```diff
@@ -48,15 +48,17 @@ class StructBuilder extends ConstantBuilder {
 
     private static final String MEMBER_MODS = "public static";
 
+    private final Declaration.Scoped structTree;
     private final GroupLayout structLayout;
     private final Type structType;
     private final Deque<String> prefixElementNames;
 
-    StructBuilder(JavaSourceBuilder enclosing, Declaration.Scoped tree,
+    StructBuilder(JavaSourceBuilder enclosing, Declaration.Scoped structTree,
```

Comment:
Should this class override `addVar` so that doc comments for getters and setters are also generated (like in the case of global vars) ?

Reply:
will check out.